subsys/bluetooth/host/Kconfig

```diff
@@ -610,6 +610,18 @@ config BT_HOST_CCM
 	  controller's AES encryption functions if available, or BT_HOST_CRYPTO
 	  otherwise.
 
+if BT_PER_ADV_SYNC
+config BT_PER_ADV_SYNC_BUF_SIZE
```

Comment:
If you have less than two or three options that depend on some other option (in this case you have just one) it's better (more compact) to use `depends on FOO` rather than a `if FOO ... endif` block.